index.html

```diff
@@ -232,17 +232,17 @@ <h1>Get Started</h1>
         I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
       </p>
 
-      <form action="/">
+      <form action="https://formspree.io/f/mqkovwwj" method="POST">
         <div class="form">
           <label for="name">Your name</label><br>
-          <input id="name" type="text" placeholder="Donice Ubaru">
+          <input id="name" type="text" name="name" placeholder="Donice Ubaru" maxlength="30" required>
           <label for="email">Your email</label><br>
-          <input id="email" type="text" placeholder="doniceubaru1@gmail.com">
+          <input id="email" type="email" name="email" placeholder="doniceubaru1@gmail.com" required>
           <label for="message">Your mesaage</label><br>
-          <textarea id="message" rows="4" cols="50" placeholder="Enter yout new message here"></textarea>
+          <textarea id="message" type="text" name="message" rows="4" cols="50" placeholder="Enter yout new message here" maxlength="500" required></textarea>
         </div>
 
-        <button type="button">Start Collaboration</button>
+        <button type="submit">Start Collaboration</button>
```

Comment:
- [ ] Kindly stick with the *Figma* design as closely as possible, it seems like your design is not matching in your work. Please have a look at the below comparison table to understand the problem in a better way through visualization, I can see that there is a slight border around the button which actually doesn't appear in the original design.

###  Design Comparison

<table>
<tr>
<th>Your Design</th>
<th>Figma Design</th>
</tr>

<tr>
<td>

![Screenshot from 2023-06-08 06-36-50](https://github.com/donice/Portfolio-Microverse/assets/66476329/9cbf8438-f6f8-444a-8956-c5cd442b29de)

</td>

<td>

![Screenshot from 2023-06-08 06-36-56](https://github.com/donice/Portfolio-Microverse/assets/66476329/1e485200-dcae-4ead-a630-2413284171b8)

</td>
</tr>
</table>


#### Requirement

![image](https://github.com/sinruidotec/Portfolio-part-2-Microverse/assets/66476329/641d0c3f-f77f-4683-a5d1-b0c8780bdbe6)